src/event_key.rs

```diff
@@ -38,5 +38,8 @@ impl EventKey {
         self.as_ref().group
     }
 
-    // TODO: add getter for is_modifier
+    // TODO: to be uncommented when the field will be generated
+    // pub fn get_is_modifier(&self) -> bool {
+    //     from_glib(self.as_ref().is_modifier)
+    // }
```

Comment:
Bitfields /o\

As the whole `GdkEvent` union is zero-allocated, you can
```
    let ptr = (&self.as_ref().group as *const u8).offset(1);
    from_glib(*ptr)
```

Reply:
I'd rather not. I prefer to just wait for the generation instead.

Reply:
We can probably wait years for that, it's far from trivial to get right for the general case

Reply:
I'm a patient man. :)

Reply:
The users might not be :P

Reply:
I'll think about it when I'll hear their complains then. :p